WcaOnRails/config/locales/en.yml

```diff
@@ -1340,7 +1340,7 @@ en:
       it is highly recommended to search for sponsors. Although WCA Delegates are volunteers and do not receive a salary for their work, organizers should consider paying compensation for their travel expenses."
     paragraph3:
       title: "Deadlines for the announcement of the competition"
-      content: "The competition should be announced at least one month before the start of the competition (see Guideline 8a4++). If there are special reasons preventing this deadline to be met, competitions may be announced up to two weeks before, at the discretion
+      content: "The competition should be announced at least one month before the start of the competition (see <a href='https://www.worldcubeassociation.org/regulations/guidelines.html#8a4++'>Guideline 8a4++</a>). If there are special reasons preventing this deadline to be met, competitions may be announced up to two weeks before, at the discretion
```

Comment:
Doesn't this need to get changed to `content_html:` for this to work? Else it will show up as plaintext, not a link.